app/code/Magento/Checkout/Block/Onepage.php

```diff
@@ -57,7 +57,7 @@ public function __construct(
         \Magento\Checkout\Model\CompositeConfigProvider $configProvider,
         array $layoutProcessors = [],
         array $data = [],
-        \Magento\Framework\Serialize\Serializer\Json $serializer = null
+        \Magento\Framework\Serialize\Serializer\JsonHexTag $serializer = null
```

Comment:
Backward incompatible changes, please inject new dependencies as the new optional parameter.
In this case actually I would change the dependency to \Magento\Framework\Serialize\SerializerInterface and configure argument as JsonHexTag in di.xml

Reply:
Updated